Tutorials/13_Geometry-Optimization/13a_Internal-Coordinates-Bmatrix.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
For the file, please do the following:

1) Group all import together at the top of a file in its own box.
2) Please do not use `v3d` code. `np.linalg.norm` works just as well.
3) `print()` does not have a space between the work and first parenthesis. Also every `print` statement need parenthesis to be Py3 comparable.
4) Please follow the PEP style guide. You can use `YAPF` to help you auto format your code.
5) I might avoid pulling from pubchem here, while cool those geometries can change due to how we match from that repo.
6) There is no need to use `fabs` unless you do not have floats.
7) Please use snake case for all functions and camel case for all classes via PEP.